python/torch_mlir/dialects/torch/importer/jit_ir/csrc/import_options.h

```diff
@@ -0,0 +1,19 @@
+//===- import_options.h -----------------------------------------*- C++ -*-===//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+// Also available under a BSD-style license. See LICENSE.
+//
+//===----------------------------------------------------------------------===//
+
+#ifndef TORCHMLIRJITIRIMPORTER_CSRC_IMPORT_OPTIONS_H
+#define TORCHMLIRJITIRIMPORTER_CSRC_IMPORT_OPTIONS_H
+
+namespace torch_mlir {
+struct ImportOptions {
```

Comment:
Can you add a comment here:

"""
Common import options across importers. We define this as a struct to avoid an unstructured proliferation of different kinds of ways to control different parts of the import process.
"""